maquetacion/application.css

```diff
@@ -1,3 +1,102 @@
+* {
```

Comment:
Add this file to the .gitignore. This is the build from the aplication.scss and it doesnt need be here.